src/main/java/com/jfrog/ide/idea/NpmProject.java

```diff
@@ -7,50 +7,39 @@
 import com.intellij.openapi.util.Key;
 import com.intellij.openapi.vfs.VirtualFile;
 import com.intellij.util.messages.MessageBus;
-import com.intellij.util.messages.impl.MessageBusImpl;
-import org.apache.commons.io.FileUtils;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import org.picocontainer.PicoContainer;
 
-import java.io.File;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-
-import static org.testng.Assert.fail;
-
 /**
- * Created by Yahav Itzhak on 25 Dec 2017.
+ * Dummy project to use for npm scan managers.
```

Comment:
I suggest adding the following to the above comment:
// Unlike maven or gradle, for npm, there there's no real project in IntelliJ. We therefore use this project.